test/integration/node-specific/distinct.test.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from 'chai';
+
+import { Collection, CommandStartedEvent, MongoClient } from '../../../src';
+import { falsyToString, falsyValues } from './comment_with_falsy_values.test';
+
+context('command distinct', function () {
+  let client: MongoClient;
+  let collection: Collection;
+  let commands: CommandStartedEvent[] = [];
+
+  beforeEach(async function () {
+    client = this.configuration.newClient({ monitorCommands: true });
+    client.on('commandStarted', e => commands.push(e));
+    await client.connect();
+    collection = await client.db('comment-falsy-values').createCollection('collection');
+    commands = [];
+  });
+
+  afterEach(async function () {
+    await collection.drop();
+    await client.close();
+  });
+
+  context('comment with falsy values', function () {
```

Comment:
why can't these be added to the list of other falsy value comment tests?

Reply:
Unlike all the other commands tested in the other file, the structure of the `distinct` command is different (the api is `distinct(string, filter?, options?)` while the others are all `<method>(options?)`).  I chose to write a standalone test file instead of modifying the existing test runner because it seemed like the more straightforward approach.  I am willing to reconsider this if you feel strongly though

Reply:
For simplicity, maybe we can just move the test to the same test file as the others just outside the loop? That way they are grouped together, but we can decide later to combine them if there are more use cases